browsers-pages.md

```diff
@@ -34,14 +34,16 @@ browser1 := rod.New().ControlURL(
 
 ## Multiple pages
 
-To control multiple pages for a browser:
+To launch multiple pages for a browser:
 
 ```go
 browser := rod.New().MustConnect()
 page1 := browser.MustPage("http://a.com")
 page2 := browser.MustPage("http://b.com")
 ```
 
```

Comment:
How about adding a section title for it?

```suggestion
## Pages not launched via Rod

```

Reply:
Well it also applies to pages launched by rod as well.